go/worker/common/committee/node.go

```diff
@@ -108,8 +108,9 @@ type Node struct {
 
 	// Mutable and shared between nodes' workers.
 	// Guarded by .CrossNode.
-	CrossNode    sync.Mutex
-	CurrentBlock *block.Block
+	CrossNode          sync.Mutex
+	CurrentBlock       *block.Block
+	CurrentBlockHeight int64
```

Comment:
This is slightly confusing as the `CurrentBlock` above refers to the runtime's block, while this block height refers to the current consensus layer block.